bylaws/3-cf-council.md

```diff
@@ -45,75 +44,74 @@ An up-to-date list of CFC members will be maintained in the `cf-council` [CONTAC
-- **Adjournment in Absence of Quorum:** If a quorum is not present at a meeting, a majority of the CFC members present may adjourn the meeting until a quorum is achieved.
-- **Reduced Quorum for Disqualified Members:** If any CFC member is disqualified from voting on a particular matter, the quorum required for that matter is reduced accordingly. Suspended CFC members (if any) are also not counted for the purposes of quorum.
+### Voting
 
-### Decision-Making Process
+The CFC prioritizes and prefers using [consensus for decision-making][consensus]. Voting is a final part in that process, and is used primarily to resolve disagreement.
 
-- **Consensus-Based Approach:** The CFC strives to make decisions based on consensus, adhering to the Lazy Consensus model outlined in [General Decision Making][consensus]. This approach emphasizes collective agreement and accommodates diverse viewpoints.
-- **Voting as a Last Resort:** If a consensus cannot be reached, the CFC resorts to voting. Decisions are then made by a simple majority of the voting CFC members present unless a supermajority is required.
-- **Supermajority Requirement:** Certain critical decisions require a supermajority vote. A supermajority is defined as at least two-thirds of the total number of CFC members.
+Most decisions can proceed with majority approval, but critical decisions require a supermajority to proceed.
 
-### Matters Requiring Supermajority Vote
+- **Significant operational decisions**: Critical decisions impacting the foundation's operations must be approved by a supermajority (2/3rds) of all CFC members. This includes suspending or revoking membership for an individual or project, or changes to the Foundation's operations.
 
-The following matters must be approved by a supermajority vote of the CFC:
+-**Policy, Bylaws or other significant structural changes:** Major policy changes and amendments to bylaws must be approved by a sumpermajority (2/3rds) of all EGC members. This includes decisions such as amending the foundation's Bylaws or Articles of Incorporation, significant alterations to the foundation's structure or operations, and changes that broadly impact member projects.
 
-- Amending the Bylaws or Articles of Incorporation.
-- Approving mergers, consolidations, or dissolution of the Foundation.
-- Changing the size or eligibility requirements of the CFC.
-- Major changes to the Foundation's structure or operations.
-- Suspending or revoking membership for an individual or project.
-- Any changes to the definition or rights of membership.
-- Other significant decisions as determined by the CFC.
+- **Overruling CFC Decisions:** In instances where the CFC makes a decision, a supermajority vote of the EGC, or a supermajority (2/3rds) of all CF members, can override the CFC's decision, ensuring a balance of power and representation within the foundation's governance structure.
```

Comment:
Sounds good.
We'll probably need a mention somewhere that the EGC or CF members may have access to the Elekto platform to spontaneously organize a vote -- for this purpose in particular, but maybe also for others.

Reply:
Yes.. we can do that w/ refinements to decision-making page and enhancements to automation to make those easier to organize.